### DISCUSSION TITLE
QUestion on Secondary Storage For New Zone

### DISCUSSION THREAD

**Discussion by midhunpjos** [apache/cloudstack#9484] [Q&A] [answered]
Hello Community,

When adding a new zone to a CloudStack deployment, is it necessary for the Secondary storage NFS to be mountable by the Management Servers? 
In my setup, the secondary storage network is entirely isolated to the zone, making the NFS share inaccessible from the management nodes.

**Comment by rajujith** [comment, marked as answer]
@midhunpjos  Secondary storage in all zones needs to be mountable from the management servers. If you can allow temporary access till the systemVM template is copied to the new secondary storage it should be sufficient.
